ktoml-core/src/commonMain/kotlin/com/akuleshov7/ktoml/exceptions/TomlSerializationExceptions.kt

```diff
@@ -0,0 +1,38 @@
+/**
+ * Internal exceptions used in the project
+ */
+
+@file:Suppress("MISSING_KDOC_TOP_LEVEL")
+
+package com.akuleshov7.ktoml.exceptions
+
+import kotlinx.serialization.SerializationException
+
+public sealed class TomlDecodingException(message: String) : SerializationException(message)
```

Comment:
If you have finally renamed it - may be make all exception's naming consistent? 
Let's select one of two variants:
1) All of them are with 'Toml' prefix
2) All of them are without a `Toml` prefix and only the base class `TomlDecodingException` contains it

Reply:
> All of them are without a `Toml` prefix and only the base class `TomlDecodingException` contains it

I prefer this, and I imagine something like `TomlEncodingException` for encoding. I'd like to make both of these as `public` for users to manipulate, while actual exception types keep `internal`.

Reply:
A further question: If there is only one file in exception folder, I think to put it out along with `Toml` and other stuff may be better, as that folder seems redundant.

Reply:
>>> I prefer this, and I imagine something like TomlEncodingException for encoding
totally agree: `TomlEncodingException`/`TomlDecodingException` sounds very good

Reply:
> A further question: If there is only one file in exceptions folder, I think to put it out along with Toml and other stuff may be better, as that folder seems redundant.

Good idea, the directory is redundant. I just automatically used some useless Java practices in Kotlin 😃

Reply:
> Hm, but that will change package name. Instead of ktoml.exceptions.BlaBlaException - you will have ktoml.BlaBlaException. May be better to have it as is...

Then I'll have `TomlEncodingException` in one file, and `TomlDecodingException` another one.